connector/MongoDBAtlasODBC.pq

```diff
@@ -69,6 +69,13 @@ shared GetConnectionString = (uri as text, database as text) as record =>
 
 // Data Source Kind description
 MongoDBAtlasODBC = [
+    TestConnection = (dataSourcePath) =>

```

Comment:
Let's add a brief comment on what this will be used for, such as for enabling gateway support.